### DISCUSSION TITLE
Payment issue: visa secure loading a blank page

### DISCUSSION THREAD

**Discussion by arsalan86** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1227] [General]
I've tried several browsers and even a VPN. This looks like something's broken at your payments processor.

![image](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/8079899/69a7013d-72f8-4bdf-b918-645926c20406)

Edit: removed PII

**Comment by MatthiasKerbl** [comment]
Hi @arsalan86,

The payment process is working without any issues for us.
Did you try different cards or PayPal?
You can also write the support of our payment provider at assist@paddle.com

Best,
Matthias
